RecoTracker/IterativeTracking/python/iterativeTkConfig.py

```diff
@@ -63,7 +63,7 @@
     "LowPtQuadStep",
     "LowPtTripletStep",
     "DetachedQuadStep",
-    "PixelPairStep",
+#    "PixelPairStep",
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant